fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/util/ExceptionUtil.java

```diff
@@ -63,11 +62,10 @@ private static RelationalException recordCoreToRelationalException(RecordCoreExc
         ErrorCode code = ErrorCode.UNKNOWN;
         if (re instanceof FDBExceptions.FDBStoreTransactionTimeoutException) {
             code = ErrorCode.TRANSACTION_TIMEOUT;
-        } else if (re instanceof RecordCoreStorageException ||
-                (re.getCause() != null && re.getCause() instanceof RecordCoreStorageException)) {
-            // Async handling can make a RecordCoreStorageException with cause of RecordCoreStorageException
-            // when 'Transaction is not active' exception.
+        } else if (re instanceof RecordContextNotActiveException || re.getCause() instanceof RecordContextNotActiveException) {
```

Comment:
There is no catch-all for `RecordCoreStorageException` anymore, which means `ErrorCode.UNKNOWN` would be returned in this case. I believe this is OK but just wanted to make sure this is the intention.